src/models/permissions/permission.entity.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  ManyToMany,
+  JoinTable,
+} from 'typeorm';
+import { UserUnionPermission } from '../user_union_permissions/user_union_permissions.entity';
+@Entity({
+  name: 'permissions',
+})
+export class Permission {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  description: string;
+
+  @Column()
+  name: string;
+
+  @ManyToMany(() => UserUnionPermission)
```

Comment:
Sorry, it my mistake in database. See https://www.figma.com/file/dDH2TCvH32jfYVj6e0ZHfd/pani-things?node-id=0%3A1
There is new relationships for users, unions, permissions and user_union_permissions.